src/core/room/room.client.ts

```diff
@@ -328,6 +328,10 @@ export class ClientRoom extends Room<WorkPlace.Client> {
     deadPlayer.bury();
   }
 
+  public async revive(deadPlayer: Player): Promise<void> {
```

Comment:
how does it revive from server side?

Reply:
because if a player has 0 or negative hp, which should be in dead. To forcefully revive the player will be a kind of way to break the game rule. I'd like to check the hp status before reviving the players

Reply:
The server emits a changePropertiesEvent to the client, which has a key `activate`, if the player died and activate is true, we will revive the player;